src/include/daos/mem.h

```diff
@@ -335,6 +356,23 @@ umem_off2ptr(const struct umem_instance *umm, umem_off_t umoff)
 	return (void *)(umm->umm_base + umoff);
 }
 
+/** Convert a pointer to offset.
+ *
+ *  \param	umm[IN]		The umem pool instance
+ *  \param	ptr[in]		The pointer to convert
+ *
+ *  \return	The offset in the PMEM pool.
+ */
+static inline umem_off_t
+umem_ptr2off(const struct umem_instance *umm, void *ptr)
```

Comment:
I have a version of this in https://github.com/daos-stack/daos/pull/406/files

It doesn't require umem_ptr2id call.  It just does arithmetic based on umm->umm_base

Reply:
OK, I will reuse your version. If your patch is landed first, then I will rebase mine.